[CW-2658] Describe LauncherData and add missing isFragile property handling in moments-sdk (#1019)
livechat/livechat-public-docs: src/pages/extending-chat-widget/chat-widget-moments/index.mdx

Default method exported by the library. It's initializing connection with chat widget. You can pass additional properties:

| param     | type    | description                                                                                                               |
| --------- | ------- | ------------------------------------------------------------------------------------------------------------------------- |

> | title     | string  | Application title, will be placed on Moment's titlebar                                                                    |

Reviewer: ```suggestion
| title     | string  | Application title; will be placed on the Moment's title bar.                                                                   |
```

> | icon      | string  | URL to Application icon, will be placed on Moments title bar                                                              |

Reviewer: ```suggestion
| icon      | string  | URL to the application icon; will be placed on the Moment's title bar.                                                              |
```

> | isFragile | boolean | When set to true, widget will require additional confirmation from user in order to close the moment. |

Reviewer 2: ```suggestion
| isFragile | boolean | When set to true, Chat Widget will require an additional confirmation from the user in order to close the moment. |
```

> | isFragile | boolean | When set to true, LiveChat Chat Widget will require an additional confirmation from the user in order to close the Moment. |

Reviewer: ```suggestion
| isFragile | boolean | When set to true, the LiveChat Chat Widget will require additional confirmation from the user in order to close the Moment. |
```

momentsSDK.close();
```

### setIsFragile

> Set `isFragile` value. It can be provided in initalization and changed later with this function.

Reviewer 2: ```suggestion
Set `isFragile` value. It can be provided as part of initialization data and changed later with this function.
```
apart from wording we could additionally reference a link to `init` payload

Parameters:

-   `value` - Boolean

```js
momentsSDK.setIsFragile(true)
```

## LauncherData

> Data passed from the Chat Widget. It is available since the moment of SDK initialization. It consists of:

Reviewer 2: using the term "moment" is a little bit unfortunate in the context of this feature 😅 
maybe something like those will also suit it well?
```suggestion
Data passed from the Chat Widget. It is available right after the SDK initialization. It consists of:
```

> -   `licenseId` - ID of the license for which Moment has been initialized

Reviewer: ```suggestion
-   `licenseId` - ID of the license for which a Moment has been initialized
```

> -   `chatId` - ID of customer's chat (returns `undefined` if no chat)

Reviewer: ```suggestion
-   `chatId` - ID of a customer's chat (returns `undefined` if there's no chat)
```

Reviewer: Sounds a bit better in terms of style and grammar
